server/api/sample_sizes.py

```diff
@@ -26,16 +26,16 @@ def sample_size_options(
     def sample_sizes_for_contest(contest: Contest):
         assert election.risk_limit is not None
         if election.audit_type == AuditType.BALLOT_POLLING:
-            cumulative_results = (
-                {choice.id: 0 for choice in contest.choices}
+            sample_results = (
+                {"round0": {choice.id: 0 for choice in contest.choices}}
```

Comment:
this seems kinda funky to me. could the math just take `None` if we haven't started the first round yet?

Reply:
or an empty dict?

Reply:
Hmm, probably. I'll mess with it and see what I can do.

Reply:
Yep, the math takes and empty dict just fine.

Reply:
awesome! do you think we could do the same for the batch results just below?

also, i just noticed that the ballot comparison case doesn't take into account `round_one` at all, which makes me think there's a bug

Reply:
Ballot comparison does, it just does it by setting the observed errors all to zero, since we haven't started yet. I think that's correct.

I'll look at batch and get back to you.

Reply:
Looks like yes, and we can even remove the conditional logic. Pushing soon.

Reply:
Done

Reply:
here's what i meant about ballot comparison: https://github.com/votingworks/arlo/pull/838

Reply:
you still need the conditional logic (read the comment at the top of the function to understand why). if there's no test that catches that we should add one

Reply:
it's probably my least favorite thing i've done in this codebase, cuz it's pretty confusing. if you can think of a better way to deal with it i'm open

two ideas i've had:
- store the sample size in the db once we start the first round, then return the stored sample size from the endpoint
- have a required `round_number` argument instead of an optional `round_one` flag

Reply:
I see. This feels like a good question for @mcchilders and/or a future PR. For now, I'm going to revert to the preexisting behavior (fixed for this PR).